CMakeLists.txt

```diff
@@ -43,10 +43,11 @@ option(${PROJECT_NAME}_CXX_WARNINGS "Compile C++ with warnings as errors" ON)
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake)
 set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY true)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Mostly for make_unique/make_shared

Reply:
make_shared is `C++11`; is std::make_unique in a critical path, because the alloc overhead of `std::unique(new ...)` might have to be incurred, depending on the wheel building.

Is there anything else that's needed?